central/metrics/custom/tracker/validator.go

```diff
@@ -65,3 +71,22 @@ func TranslateConfiguration(config map[string]*storage.PrometheusMetrics_Group_L
 	}
 	return result, nil
 }
+
+// ValidateConfiguration validates and translates a metric group configuration.
+func ValidateConfiguration(metricsGroup *storage.PrometheusMetrics_Group, currentMetrics MetricsConfiguration, labelOrder map[Label]int) (*Configuration, error) {
```

Comment:
I think the naming between `TranslateConfiguration` and `ValidateConfiguration` is confusing here.

* ambiguous names `MetricsConfiguration` and `Configuration`
* both functions seem to do translation and validation

How about something along the lines of `GroupLabelsToMetricsConfiguration` and `MetricsGroupToConfiguration`? Perhaps even put them into a `convert` package to it reads `convert.xxx`.